v3/how-to-guides/08-parachains/f-rococo/index.mdx

```diff
@@ -0,0 +1,169 @@
+In the long-term, the plan is to make it a community-driven process via Rococo's governance framework and remove the need to rely on a sudo key.
+
+For reference: the `AssignSlotOrigin` origin can allot a temporary slot by doing the following:
+
+1. Go to the [Polkadot-JS Apps for Rococo](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-rpc.polkadot.io#/extrinsics), under `Developer -> Extrinsics`.
+1. Select the account with ROC balance that you wish to use to submit a transaction with. You will need at least 5 ROCs.
+1. From the `assignedSlots` pallet, select the `assignTempParachainSlot` function.
+1. Insert a reserved paraID with associated parathread and select `Current` for the `LeasePeriodStart`.
+1. Sign and submit the transaction.
+
+If the current slots are full, the parathread will be assigned and upgraded to a parachain with the next available slot.
+
+
+🎉**_Congratulations!_**🎉
+
+With your slot activated, you're now able to test your parachain on the Rococo testnet!
+Please do note that when your temporary slot's lease ends, the parachain is **automatically downgraded** to a parathread.
+Registered and approved slots are **cycled through automatically** in a round-robin fashion, so you will expect to come back online as a parachain from time to time.
+
+Given a period lease duration of 1 day, the current settings for assigned parachain slots on Rococo are (subject to change, based on the needs of the community):
+
+- **Permanent slot lease duration**: 1 year (365 days)
+- **Temporary slot lease duration**: 3 days
+- **Maximum number of permanent slots**: up to 25 permanent slots
+- **Maximum number of temporary slots**: up to 20 temporary slots
+- **Maximum temporary slots allocated per leased period**: up to 5 temporary slots per 3-day temporary lease periods
+
+Here's an example with parathreads `3000`-`3010` registered for a temporary slot (including chain spec and Wasm blob), starting at lease period 0:
+
+- On the first lease day (period 0), parathreads `3000`-`3005` will onboard as parachains and remain onboarded for 3 lease periods (3 days).
+  At the end of lease period 3, parachains with IDs `3000` - `3005` will automatically be offboarded and will become parathreads again.
+- At the start of the 4th lease day (period 4), the next batch of parathreads registered with IDs `3006`-`3010` will be onboarded as parachains for 3 days.
+  Then by the end of lease period 6, parachains with IDs `3006` - `3010` will be offboarded and will become parathreads again.
+- At the start of the 7th lease day (period 7), the next batch of registered parathreads to onboard will be `3000`-`3005`, which will automatically get onboarded as parachains for 3 days. And the cycle repeats.
+
+If you are interested in discovering how this lease management is done, you can have a look at the [source code](https://github.com/paritytech/polkadot/blob/master/runtime/common/src/assigned_slots.rs).
```

Comment:
@al3mart here is the basics of the slot detials, but more about the actual logic could be useful if you see fit. If you think this is detailed enough, then OK. We can do a follow up PR if you like as well. I would think we are good to approve once you or Sacha give a final pass. Feel free to squash merge this PR once approved

Reply:
As I see changes coming to this precise pallet I would say following with a PR could be more handy.